api/api_test.go

```diff
@@ -913,9 +913,8 @@ func TestAPI_Headers(t *testing.T) {
 	  policy = "read"
 	}
 	`))
-	// ACL support is disabled
+	// Legacy ACL support is deprecated.
```

Comment:
What do you think about simplifying this to a single statement?

```go
_, err = c.ACL().RulesTranslate(nil)
require.Error(t, err)
```

I think it's probably also a good idea to just remove the comment about legacy deprecation, since we're actually removing legacy ACLs.

An alternative would be to remove the tests for `ACL.RulesTranslate` and `ACL.RulesTranslateToken` into a dedicated test...

Reply:
added separate tests for the deprecated paths